packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -359,7 +360,10 @@ export class SelectionZone extends React.Component<ISelectionZoneProps, ISelecti
             }
           }
           break;
-        } else if (this._hasAttribute(target, SELECTION_INVOKE_ATTRIBUTE_NAME)) {
+        } else if (
```

Comment:
Can you make this behavior opt-in in the `SelectionZone`? I don't want to accidentally break existing partner apps that expect a different behavior right now.

Reply:
Have added a new prop `enableTouchInvocationTarget`